Source/WebInspectorUI/UserInterface/Views/Main.css

```diff
@@ -579,6 +578,11 @@ body[dir=rtl] [dir=ltr] .go-to-arrow {
     inset-inline-end: 0.5em;
 }
 
+.accessibility-settings-content {
+    display: flex;
+    flex-flow: column;
```

Comment:
NIT: id prefer to use `flex-direction` if you're not setting a `flex-wrap` value